### DISCUSSION TITLE
mirth system time

### DISCUSSION THREAD

**Discussion by MrTianFei** [nextgenhealthcare/connect#4783] [Q&A] [answered]
The current time output by mirth is inconsistent with my computer's system time，my computer's system time was 2021-10-13
![(6{56 D`)URS5AU}TI)$XIG](https://user-images.githubusercontent.com/86398277/137088130-68038106-2990-4815-b013-211b31fd5e5a.jpg)

![BL0W@8F`D)U6U9WL84K6$YG](https://user-images.githubusercontent.com/86398277/137088254-695d3b15-6dca-409b-ae70-d9d6b066e39d.jpg)

**Comment by tonygermano** [comment, marked as answer]
You used minutes where you expected months. Try `yyyy-MM-dd hh:mm:ss`

**Reply by MrTianFei** [reply]
Yes, I made a mistake
